Library/Formula/google-sql-tool.rb

```diff
@@ -1,19 +1,17 @@
-require 'formula'
-
 class GoogleSqlTool < Formula
-  homepage 'https://developers.google.com/cloud-sql/docs/commandline'
-  url 'http://dl.google.com/cloudsql/tools/google_sql_tool.zip'
-  sha1 '67111a75a1c37b34527b3954795b179f62e88632'
-  version 'r10'
+  homepage "https://developers.google.com/cloud-sql/docs/commandline"
+  url "https://dl.google.com/cloudsql/tools/google_sql_tool.zip"
+  sha256 "b7e993edab12da32772bfa90c13999df728f06792757c496140d729d230b03c3"
+  version "r10"
 
   def install
     # Patch script to find jar
-    system "chmod +w google_sql.sh"
-    inreplace 'google_sql.sh',
+    chmod 0755, "google_sql.sh"
+    inreplace "google_sql.sh",
       'SQL_SH_DIR="$(cd $(dirname $0); pwd)"',
       "SQL_SH_DIR=\"#{libexec}\""
 
-    libexec.install %w(google_sql.sh google_sql.jar)
-    bin.install_symlink libexec+('google_sql.sh') => 'google_sql'
+    libexec.install ["google_sql.sh", "google_sql.jar"]
+    bin.install_symlink libexec+("google_sql.sh") => "google_sql"
   end
 end
```

Comment:
If you're feeling lovely: could a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!